### ISSUE TITLE
object-shorthand rule errors on getters and setters

### ISSUE THREAD

**Issue by bdougherty** [eslint/eslint#2563]
The following code generates an error with `"object-shorthand": [2, "always"]`:

```js
var x = {
    get y() {},
    set y(z) {}
};
```

Did a little digging and `node.shorthand === false` for those, so maybe the rule should exclude nodes where are `node.kind === "get"` or `node.kind === "set"`.

Using version 0.21.1.

**Comment by michaelficarra** [comment]
Sounds good to me.

**Comment by nzakas** [comment]
Makes sense. In the future, please be sure to include the exact output from ESLint when reporting an issue.

**Comment by michaelficarra** [comment]
@bdougherty do you want me to open a PR for you with the referenced commit?
